dependency/controller.py

```diff
@@ -50,40 +49,21 @@ def get_dependencies(workspace_id: str):
         else:
             ids = ids.split(',')
             dependencies = service.get_multiple_by_id(ids)
-    except DBException as e:
-        return jsonify({
-            "status": "failure",
-            "error": e.message
-        }), 400
-
-    return jsonify({
-        'status': 'success',
-        'result': [responsify_dependency(dependency) for dependency in dependencies],
-    }), 200
-
-@blueprint.get('/<dependency_id>')
-def get_dependency(workspace_id: str, dependency_id: str):
-    try:
-        dependency = service.get_by_id(dependency_id)
-    except DBException as e:
-        return jsonify({
-            "status": "failure",
-            "error": e.message
-        }), 400
 
-    return jsonify({
-        'status': 'success',
-        'result': responsify_dependency(dependency),
-    })
+        return responses.success_response("get_dependencies",
+            [responsify_dependency(dependency) for dependency in dependencies], object_name="dependency")
+    except ValidationException as e:
+        return responses.validation_error_response(message=str(e), type=type(e).__name__)
+    except Exception as e:
+        return responses.unknown_error_response(message=str(e), type=type(e).__name__)
```

Comment:
re: an earlier comment, I don't think we want to expose unexpected exceptions via the API (not that security matters much for our app but I can't see any use that front-end would have for them either other than to file a bug-report - but as a dev they could just look at the console)